pkg/apic/servicebody.go

```diff
@@ -96,37 +94,24 @@ func (s *ServiceBody) GetCredentialRequestDefinitions() []string {
 	return s.credentialRequestPolicies
 }
 
-func (s *ServiceBody) setAccessRequestDefintion(accessRequestDefinition *mv1a.AccessRequestDefinition) (*mv1a.AccessRequestDefinition, error) {
+func (s *ServiceBody) setAccessRequestDefinition(accessRequestDefinition *mv1a.AccessRequestDefinition) (*mv1a.AccessRequestDefinition, error) {
 	s.accessRequestDefinition = accessRequestDefinition
 	return s.accessRequestDefinition, nil
 }
 
-// GetAccessRequestDefintion -
-func (s *ServiceBody) GetAccessRequestDefintion() *mv1a.AccessRequestDefinition {
+// GetAccessRequestDefinition -
+func (s *ServiceBody) GetAccessRequestDefinition() *mv1a.AccessRequestDefinition {
 	return s.accessRequestDefinition
 }
 
-func (s *ServiceBody) createAccessRequestDefintion() error {
+func (s *ServiceBody) createAccessRequestDefinition() error {
 	oauthScopes := make([]string, 0)
 	for scope := range s.GetScopes() {
 		oauthScopes = append(oauthScopes, scope)
 	}
 	if len(oauthScopes) > 0 {
 		// sort the strings for consistent specs
-		sort.Strings(oauthScopes)
-		_, err := provisioning.NewAccessRequestBuilder(s.setAccessRequestDefintion).
-			SetRequestSchema(
-				provisioning.NewSchemaBuilder().
-					AddProperty(
-						provisioning.NewSchemaPropertyBuilder().
-							SetName("scopes").
-							SetLabel("Scopes").
-							IsArray().
-							AddItem(
-								provisioning.NewSchemaPropertyBuilder().
-									SetName("scope").
-									IsString().
-									SetEnumValues(oauthScopes)))).Register()
+		_, err := provisioning.NewAccessRequestBuilder(s.setAccessRequestDefinition).Register()
```

Comment:
Still creating the ARD here, just not adding any scope stuff.

Reply:
should I basically follow this as part of the marketplacemigration.go (func registerAccessRequestDefinition) as well? Meaning remove the scopes?

Reply:
Scopes need to be removed wherever they are being added. If there is another spot I missed I can remove it, just let me know.

Reply:
Ill update it in my branch.  Working on an issue with mp migration. So easy enough for me to update mine

Reply:
I have it updated in my branch and it is working, so we good. I approved.